.env.logion

```diff
@@ -0,0 +1,8 @@
+# Connect to a local node.
+#
+# For more information on how to use .env files and environment variables
+# consult the Configuration section in the README.
+
+SAS_SUBSTRATE_WS_URL=ws://127.0.0.1:9944
+NODE_TLS_REJECT_UNAUTHORIZED=0
+SAS_EXPRESS_PORT=8081
```

Comment:
Add `SAS_SUBSTRATE_TYPES` here as well (instead of setting it in `package.json` script entry)?

Reply:
I tried but failed - the env variable `$PWD` is not evaluated. If you have some trick to workaround this...

Reply:
Nothing obvious that I can think about. Leave unchanged then, it's fine.